plugins/doc_fragments/junos.py

```diff
@@ -11,7 +11,65 @@
 class ModuleDocFragment(object):
 
     # Standard files documentation fragment
-    DOCUMENTATION = r"""options: {}
+    DOCUMENTATION = r"""options:
```

Comment:
We absolutely do not want to return this to every module. Just package and scp should be fine.